api/main.py

```
@@ -32,6 +64,97 @@
+            You are an AI assistant specialized in professional communication. 
+            Rewrite the following sentence to correct its grammar and make it sound more natural and smooth, 
+            as if written by a native speaker. 
+            Do not include any additional explanations or text, just the final, polished sentence: {text_input.text}
+            """)
+        print(f"Generated prompt: {prompt}")
+
+        response = model.generate_content(prompt, generation_config={"temperature": 0})
+        
+        if response.prompt_feedback.block_reason:
+            print(f"Content blocked: {response.prompt_feedback.block_reason}")
+            raise HTTPException(status_code=400, detail=f"Content blocked: {response.prompt_feedback.block_reason}")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)